spec/lib/govuk_elements_form_builder/form_builder_spec.rb

```diff
@@ -91,136 +79,106 @@ class TestHelper < ActionView::Base; end
       let(:params) { [:email, class: custom_class] }
 
       it 'adds custom class to the input' do
-        expect(input.classes).to include(custom_class)
+        expect(tag.classes).to include(custom_class)
       end
     end
 
-    context 'has an input_prefix option' do
-      let(:prefix) { '£' }
-      let(:params) { [:email, input_prefix: prefix] }
+    context 'pass a label parameter with text and size' do
+      let(:custom_label) { "Your client's email" }
+      let(:params) { [:email, label: { text: custom_label, size: :m }] }
```

Comment:
Again - this ties the shared example to only test an "email" attributes. So I'd suggest:
```ruby
let(:params) { [attribute.to_sym, label: { text: custom_label, size: :m }] }
```